changelog/v0.44.1/cve-go-1-24-6.yaml

```
@@ -0,0 +1,13 @@
+changelog:
```

Any reason we're not including the go DEPENDENCY_BUMP changelog in this repo, as we did in e.g. k8s-utils? Maybe it's getting pulled in automatically? 

```suggestion
changelog:
- type: DEPENDENCY_BUMP
  dependencyOwner: golang
  dependencyRepo: go
  dependencyTag: v1.24.6
  description: "Update Go to 1.24.6"
  issueLink: https://github.com/solo-io/solo-projects/issues/8512
  resolvesIssue: false
```